code/front-end/Satellite GUI.md

```diff
@@ -0,0 +1,35 @@
+# Satellite GUI
+
+Satellite GUI is a [vue.js](https://vuejs.org/) + [TypeScript](https://www.typescriptlang.org/) front-end codebase that provides an UI interface for users to interact with the network.
+
```

Comment:
awesome you are writing a doc on this! thank you.

one thing i like to do for docs is to add a `Purpose` section at the top of the doc so people reading it know if they are reading the correct thing